File: 백준/Bronze/29790. 임스의 메이플컵/임스의 메이플컵.py
N, U, L = map(int, input().split())

sign = 0

if N >= 1000 : 
    sign = 1
else : 
    print('Bad')


if sign == 1 and (U >= 8000 or L >= 260) :
    sign += 1

if sign == 1 : 
    print('Good')
elif sign == 2 : 
    print('Very Good')

File: 백준/Bronze/29790. 임스의 메이플컵/임스의 메이플컵.cs
using System;

class Program
{
    static void Main()
    {
        string[] inputs = Console.ReadLine().Split();
        int N = int.Parse(inputs[0]);
        int U = int.Parse(inputs[1]);
        int L = int.Parse(inputs[2]);
        int sign = 0;

        if (N >= 1000)
        {
            sign = 1;
        }
        else
        {
            Console.WriteLine("Bad");
        }

        if (sign == 1 && (U >= 8000 || L >= 260))
        {
            sign++;
        }

        if (sign == 1)
        {
            Console.WriteLine("Good");
        }
        else if (sign == 2)
        {
            Console.WriteLine("Very Good");
        }
    }
}
